src/main/java/org/elasticsearch/search/suggest/phrase/PhraseSuggestionBuilder.java

```diff
@@ -182,20 +182,13 @@ public PhraseSuggestionBuilder collateQuery(String collateQuery) {
     /**
      * Sets a filter used for filtering out suggested phrases (collation).
      */
+    @Deprecated
     public PhraseSuggestionBuilder collateFilter(String collateFilter) {
         this.collateFilter = collateFilter;
         return this;
     }
 
     /**
-     * Sets routing preferences for executing filter query (collation).
-     */
-    public PhraseSuggestionBuilder collatePreference(String collatePreference) {
```

Comment:
FYI this removal broke some clients, see https://discuss.elastic.co/t/elasticsearch-api-instability/2574/3. Should we add it back in 1.6.1, even as a no-op?